orb-version/orb.yml

```
@@ -0,0 +1,32 @@
+version: 2.1
+
+description: >
+  Sets github version info in a text file.
+display:
+  source_url: https://github.com/qlik-oss/ci-tools/orb/github-version
+
+commands:
+  set-version:
+    description: Generate SemVer format *.*.* or *.*.*-shortCommitID
```

```suggestion
    description: Generate SemVer format *.*.* or *.*.*-x-g<shortCommitID>
```

The git describe format is *.*.*-<numberOfCommitsSinceTag>-g<shortCommitID>